microprofile/lra/participant/src/main/java/org/wildfly/extension/microprofile/lra/participant/MicroProfileLRAParticipantSubsystemModel.java

```diff
@@ -13,7 +13,7 @@
  * @author Paul Ferraro
  */
 enum MicroProfileLRAParticipantSubsystemModel implements SubsystemModel {
-    VERSION_1_0_0(1),
+    VERSION_2_0_0(2),
```

Comment:
The old enum value should be retained and a new one added.

Needing the old one is relevant when it comes to domain mode transformers, which aren't needed here, but regardless this should follow the general pattern and have an enum value for each API version that exists.